programs/programs/il/pe/member.py

```
@@ -59,6 +60,38 @@ class IlAabd(PolicyEngineMembersCalculator):
+
+    Both programs share the same eligibility logic through PolicyEngine's il_fpp_eligible variable.
+    """
+
+    pe_name = "il_fpp_eligible"
+    pe_inputs = [
+        *pe_dependency.irs_gross_income,
+        member_dependency.TaxUnitHeadDependency,
+        member_dependency.TaxUnitSpouseDependency,
+        household_dependency.IlStateCodeDependency,
+        member_dependency.PregnancyDependency,
+    ]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)